packages/wallets/binance-wallet/src/index.ts

```diff
@@ -0,0 +1,16 @@
+import { WalletAdapterType } from '@reef-knot/types';
+import { WalletIcon } from './icons/index.js';
+import { getWagmiConnectorV2 } from '@binance/w3w-wagmi-connector-v2';
+
+export const id = 'binanceWallet';
+export const name = 'Binance Web3 Wallet';
+
+const binanceWalletConnector = getWagmiConnectorV2();
```

Comment:
https://www.npmjs.com/package/@binance/w3w-wagmi-connector-v2?activeTab=code
Looking at this code maybe it makes sense to call `getWagmiConnectorV2()` inside `BinanceWeb3Wallet`?

Reply:
So that binance checks can be deterred and code not called at all if wallet is not included into wallet list?

Reply:
Maybe, I will look at this approach. I was referencing their docs here https://developers.binance.com/docs/binance-w3w/quick-start#integration-with-wagmi-v2

Reply:
@Jeday
Moved the call inside the wallet adapter